compiler/rustc_target/src/asm/avr.rs

```diff
@@ -36,15 +39,20 @@ impl AvrInlineAsmRegClass {
     ) -> &'static [(InlineAsmType, Option<&'static str>)] {
         match self {
             Self::reg => types! { _: I8; },
-            Self::reg_xyz => types! { _: I16; },
+            Self::reg_upper => types! { _: I8; },
+            Self::reg_pair => types! { _: I16; },
+            Self::wreg => types! { _: I16; },
+            Self::ptr => types! { _: I16; },
```

Comment:
This should probably have `reg` somewhere in the name for consistency with the others. `reg_ptr`? `ptr_reg`?

Reply:
Not to mention how strange a bare `ptr` looks...  Will update to `reg_ptr`.